examples/java/veeam-vm-backup/veeam-vm-backup.yml

```diff
@@ -0,0 +1,13 @@
+version: 1.0
+provider:
+  name: openfaas
+  gateway: https://VEBA_FQDN_OR_IP
+functions:
+  veeam-vm-backup:
+    lang: java8
+    handler: ./vm-backup
+    image: vmware/veeam-vm-backup
```

Comment:
Can you please change this to `vmware/veba-java-veeam-vm-backup:latest`

Reply:
This is done in the updated files.